pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -347,8 +347,11 @@ func (r *Replica) RangeFeed(
 		}
 	}
 
+	// The caller is responsible for setting the admission header
+	// to NormalPriority for any system table rangefeed.
+	isSystemRangefeed := admissionpb.WorkPriority(args.AdmissionHeader.Priority) > admissionpb.BulkNormalPri
```

Comment:
nit: should we also add a comment here (https://github.com/cockroachdb/cockroach/blob/13f8eb8126aa2bb1b2ed76948d1b8dc01b5919ac/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go#L635-L638) to clarify that only system ranges are allowed to use a priority higher than BulkNormalPri? (to prevent future issues where someone might introduce a different priority, breaking the assumption that anything above BulkNormalPri is reserved for system rangefeeds.)